priv/test_repo/migrations/20221011201645_install_carbonite_foo.exs

```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: Apache-2.0
+
+defmodule Carbonite.TestRepo.Migrations.InstallCarboniteAlternateTestSchema do
+  use Ecto.Migration
+
+  def up do
+    Carbonite.Migrations.up(1, carbonite_prefix: "alternate_test_schema")
+    Carbonite.Migrations.up(2, carbonite_prefix: "alternate_test_schema")
+    Carbonite.Migrations.up(3, carbonite_prefix: "alternate_test_schema")
+    Carbonite.Migrations.up(4, carbonite_prefix: "alternate_test_schema")
+    Carbonite.Migrations.up(5, carbonite_prefix: "alternate_test_schema")
+     end
```

Comment:
the indentation is off here, I wonder if we're missing something in `.formatter.exs`?